contrib/emojisum-py/src/emojisum/__init__.py

```diff
@@ -0,0 +1,101 @@
+import hashlib
+import json
+import mmap
+import os
+import re
+import sys
+from pathlib import Path
+from typing import NamedTuple, Optional
+
+from rich.console import Console
+
+SHA1SUM_PATTERN = re.compile(r"(?P<digest>\S*)\s+(?P<path>.*)")
+OPENSSL_SHA1_PATTERN = re.compile(r"SHA1\((?P<path>.*)\)=\s+(?P<digest>.*)")
+OUTPUT_PATTERNS = [SHA1SUM_PATTERN, OPENSSL_SHA1_PATTERN]
+
+
+class Sha1Result(NamedTuple):
+    hex_digest: str
+    path: str
+
+
+def load_table():
+    with (Path(__file__).parent / "emojimap.json").open() as f:
```

Comment:
won't this need the json installed with the python package?

Reply:
Yep, which it is. See `MANIFEST.in`.

Reply:
:+1: